test/integration/invalid_addr.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+echo "1..5"
+
+stack exec -- vaultenv \
```

Comment:
Can you add some comments to these test cases to explain:

 - What is being tested
 - Why it is being tested